### PR TITLE
[Azure ABAC] Custom security attribute with allowed values

### PR DESCRIPTION
**Pull request by rolyon**

### CODE REVIEW

File: api-reference/beta/api/customsecurityattributedefinition-update.md [hunk 1]
```diff
@@ -117,7 +117,57 @@ Content-Type: application/json
...
++ Attribute set: `Engineering`
++ Attribute: `Project`
++ Attribute data type: Collection of Strings
++ Update predefined value: `Baker`
++ New predefined value: `Skagit`
+
+> [!NOTE]
+> For this request, you must add the following to the header:<br/>
+> Key: odata-version<br/>
+> Value: 4.01
```
Comment (FaithOmbongi) [inline api-reference/beta/api/customsecurityattributedefinition-update.md:133]: Hi @rolyon, is there a reason why only this scenario requires the header?
Also, should we then add this info to the **Request headers** sections of both the Update and POST operations?

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @rolyon for your work on this PR. Let a Q about the purpose of the header and scenarios where it's required. Let's clarify this internally.

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Review feedback left here: https://github.com/microsoftgraph/microsoft-graph-docs/pull/15528#pullrequestreview-862122095

File: api-reference/beta/api/customsecurityattributedefinition-update.md (hunk 1 above)
Comment (loradule) [reply api-reference/beta/api/customsecurityattributedefinition-update.md:133]: Hi Faith,

The update implementation leverages a feature called delta update that requires OData 4.01. This is specific to update, the POST does not need this particular header. Hope this helps!

**Review by FaithOmbongi — COMMENTED** [review]
Thank you @rolyon, @loradule for the clarification. Left inline suggestions based on the feedback.

**Comment by rolyon** [comment]
Hi @FaithOmbongi 
See doc updates.
thanks

**Review by FaithOmbongi — APPROVED** [review]
Thank you @rolyon for your work on this PR and for bringing this new capability to the attention of our customers.

**Pull request merged**